### DISCUSSION TITLE
Correlation : Fetching the response in a random order

### DISCUSSION THREAD

**Discussion by JayasimhaNagaraj** [artilleryio/artillery#973] [Artillery]
Hi All,

Would like to know if its feasible to correlate based on Left and Right boundaries. If yes, I am stuck with a scenario wherein the correlation is returning multiple values and I need to pick any of these values randomly during every iteration.
Ex :We have a request which provides the list of student names. We need to correlate this and fetch each student name randomly for every iteration.
Is there any concept of Ordinal(similar to that of loadrunner) available in artillery?

Any pointers around this would be really helpful.

Thank you.

**Comment by hassy** [comment]
Where does the list of students (to use your example) come from? e.g. is it read from a CSV file, or returned from an HTTP request in the scenario?

**Reply by JayasimhaNagaraj** [reply]
Hi Hassy,
Thanks for you response.
It's returned from an HTTP request in the scenario

**Reply by kshamajk** [reply]
Hi Hassy,

Wanted to know if the above scenario is achievable from Artillery i.e., to extract based on the left and right boundaries from an HTTP response.
If yes, I have scenario where, I have left and right boundaries same for multiple values. Based on the order/occurrence number I would extract it. Can this be achieved using Artillery ?

E.g, Value="A" & Input="Hi" /n
       Value="A" & Input="Hello" /n

In this case, my left boundary will be "**Value="A" & Input="**" and right boundary would be "**/n**"

Thanks
